main/main.go

```diff
@@ -760,32 +754,33 @@ func setColorMode() {
 	}
 }
 
+// FIXME: Update this to work with the new way of handling colour schemes.
 func setSchemeSettings() {
-	for i, s := range scheme.Settings {
-		var (
-			fi = defaultFg
-			bi = defaultBg
-		)
-		if fg, ok := s.Settings["foreground"]; ok {
-			fi = palLut(fg)
-			if i == 0 {
-				defaultFg = fi
-			}
-		}
-		if bg, ok := s.Settings["background"]; ok {
-			bi = palLut(bg)
-			if i == 0 {
-				defaultBg = bi
-			}
-		}
-	}
+	// for i, s := range scheme.Settings {
+	// 	var (
+	// 		fi = defaultFg
+	// 		bi = defaultBg
+	// 	)
+	// 	if fg, ok := s.Settings["foreground"]; ok {
+	// 		fi = palLut(fg)
+	// 		if i == 0 {
+	// 			defaultFg = fi
+	// 		}
+	// 	}
+	// 	if bg, ok := s.Settings["background"]; ok {
+	// 		bi = palLut(bg)
+	// 		if i == 0 {
+	// 			defaultBg = bi
+	// 		}
+	// 	}
+	// }
 }
 
 func createNewView(filename string, window *backend.Window) *backend.View {
 	v := window.OpenFile(filename, 0)
 
 	v.Settings().Set("trace", true)
-	v.SetSyntaxFile(path.Join(backend.LIME_PACKAGES_PATH, "go.tmbundle", "Syntaxes", "Go.tmLanguage"))
+	v.SetSyntaxFile(path.Join("..", "packages", "go-tmbundle", "Syntaxes", "Go.tmLanguage"))
```

Comment:
Also we have syntax detection based on file type from https://github.com/limetext/backend/commit/9d7f5428022bda8c896f2412d00394a5e31cfe61, so no need to set the syntax manually.